lib/livebook_web/live/session_live.ex

```diff
@@ -565,33 +578,42 @@ defmodule LivebookWeb.SessionLive do
         _url,
         %{assigns: %{live_action: :catch_all}} = socket
       ) do
```

Comment:
Note that we still want `assert_live_action_access!(socket)` here. In practice it's probably not necessary, but it's better to have it on all `handle_params` just to be sure/consistent :)

Reply:
sounds good!